### ISSUE TITLE
    "@prisma/client": "^5.19.1" got inserted to "dependencies" in package.json file, causing trouble

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/wasp#2330]
This happened to me while testing out waspc/examples/todoApp with new TS SDK.

Maybe it was caused by `wasp ts-setup` that comes just for purpose of TS SDK?

Or maybe it was caused by recent updates we did to Prisma and now Prisma is doing this as a new beahviour?

Anyway, we need to make sure this doesn't happen, as it caused trouble due to `@prisma/client` ending up going to 5.20 due to `^` and then we have a missmatch between 5.20 client and 5.19 the rest of prisma (you get warning in the CLI).

**Comment by amirseyedian** [comment]
I was able to temporarily fix the issue by downgrading to `0.14.2` from `0.15.1`.

**Comment by infomiho** [comment]
@amirseyedian Do you know which steps led to you having the problem? Ideally, we'd fix this in `0.15.1` rather then requiring a downgrade.

**Comment by amirseyedian** [comment]
Sure I transported my project to a new device, installed wasp with the script (which installed the latest version) ran `wasp clean` and `npm i` then `wasp start`.
>Wasp requires package "prisma" to be version "5.19.1" in package.json.

I was also able to reproduce the error by creating new projects using templates
`wasp new test -t saas` and `wasp start` resulted in the same error.
I tried to mess around with the prisma dependency version but that didn't make any difference.

**Comment by Martinsos** [comment]
@amirseyedian I just tried the following on my machine:
```
cd ~/test/
wasp new TestSaas -t saas
cd TestSaas/app
wasp start
```
Then I checked `package.json`, but I don't see `@prisma/client` being inserted under its dependencies, meaning I failed to replicate it.

But for you, these exact steps brought you to this error? Did I miss some step?

Also, can you tell us a bit more about your system maybe? Operating system? Node / npm version?

**Comment by amirseyedian** [comment]
I upgraded to 0.15.1 again just to dublecheck and you are correct the error no longer occurs while creating new projects from template however the error still persists regarding my previous project.

**Comment by jeremysltn** [comment]
I had this error after I tried to migrate my 0.14.x project by just changing the wasp version in main.wasp and reinstalling wasp.

Fixed it by following the [Migration from 0.14.0 to 0.15.0 doc](https://wasp-lang.dev/docs/migration-guides/migrate-from-0-14-to-0-15). However, _prisma_ should be in _devDependencies_ and not in _dependencies_.

**Comment by infomiho** [comment]
@jeremysltn You are quite right! The migration docs are wrong and the `prisma` deps should be in the `devDependencies`.

**Comment by cprecioso** [comment]
I don't think this is happening anymore
